<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>魔客砖头--仅限内部使用</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>软件安装</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabBackup">
     <attribute name="title">
      <string>备份系统</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_backup_plan">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>551</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_backup">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开始备份</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowserError">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>551</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_restore">
     <attribute name="title">
      <string>恢复系统</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
